p2p/src/authenticated/actors/tracker/set.rs

```diff
@@ -69,15 +70,24 @@ impl<P: Array> std::ops::Index<usize> for Set<P> {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use commonware_utils::{array::U64, BitVec};
+    use commonware_cryptography::{ed25519, PrivateKeyExt, Signer};
+    use commonware_utils::BitVec;
     use std::collections::HashSet;
 
-    fn create_test_peers() -> Vec<U64> {
-        vec![U64::new(3), U64::new(1), U64::new(2)]
+    fn create_test_peers() -> Vec<ed25519::PublicKey> {
+        vec![
+            ed25519::PrivateKey::from_seed(3).public_key(),
+            ed25519::PrivateKey::from_seed(1).public_key(),
+            ed25519::PrivateKey::from_seed(2).public_key(),
+        ]
     }
 
-    fn expected_sorted_peers() -> Vec<U64> {
-        vec![U64::new(1), U64::new(2), U64::new(3)]
+    fn expected_sorted_peers() -> Vec<ed25519::PublicKey> {
```

Comment:
While not strictly necessary to convert from `Array` to `PublicKey`, I did so for consistency although it added some minor complication in the "sorting".